homeassistant/components/reolink/host.py

```diff
@@ -28,6 +29,7 @@
 from homeassistant.helpers.dispatcher import async_dispatcher_send
 from homeassistant.helpers.event import async_call_later
 from homeassistant.helpers.network import NoURLAvailableError, get_url
+from homeassistant.util.dt import DEFAULT_TIME_ZONE, now
```

Comment:
Always use `utcnow()` when you don't care about timezones, so DST is not impacting your code

Reply:
Even better would just be to use `time.time()` and not care about timezones. Then you can also set to never-checked by setting to 0

Reply:
Thanks for the suggestion, using time.time() is indeed cleaner. I implemented that now.